### ISSUE TITLE
Hotkey to simulate Mousedown on screen-capture screen.

### ISSUE THREAD

**Issue by Murilirum** [ShareX/ShareX#926]
Right now it requires that I Mousedown in order to initiate a screenshot, I'd like to be able to use a keyboard only option with capturing some smaller, pixel-sensitive elements on my screen without having to switch from the arrow keys, to the mouse for movement and then back to arrow keys. Not an awful experience on it's own, so this is merely a Quality-of-life type of enhancement.

![](http://i.imgur.com/B5P5lvi.png)

In short, I'm targeting the [Hold Left Click] option and requesting that a keybind option (of some kind) be added if it hasn't already.

**Comment by davidruhmann** [comment]
Created one possible implementation with <kbd>Insert</kbd> and <kbd>End</kbd>.  Opinions on those keys?

**Comment by Jaex** [comment]
You used <kbd>Insert</kbd> for start and again <kbd>Insert</kbd> for cancel it. But in my opinion <kbd>Insert</kbd> should be finisher too same as mouse. Example we down <kbd>Left click</kbd> to start up <kbd>Left click</kbd> to capture it. But using <kbd>Right click</kbd> to cancel it which should be <kbd>End</kbd>.

And in F1 tips it tells "Stop editing" but people can think it is canceling so it should be changed to something which should tell user it will be captured I think?

**Comment by Murilirum** [comment]
Is there any reason that would prohibit the holding-down of the tab key while moving with the arrow keys? Just trying to think in the sense of natural placement of either hand on the keyboard.

**Comment by Jaex** [comment]
Tab key already in use.

**Comment by Murilirum** [comment]
What about the <kbd>s</kbd> only while the selection interface is active?

**Comment by Jaex** [comment]
Remembering alphabet keys can be more difficult for people than using <kbd>Insert</kbd> key I think, also need to hold <kbd>Ctrl</kbd> in left hand constantly to speed it up anyway.

So my suggestion is:

<kbd>Insert</kbd> = Start region selection
<kbd>Insert</kbd> again = Finish selection
<kbd>Delete</kbd> = Cancel selection / Cancel capture

**Comment by davidruhmann** [comment]
Reason I choose <kbd>Insert</kbd> was I liked a one handed solution.

>Example: Move to desired location with arrow keys, reach above arrow keys with same hand and press insert, move back to arrow keys to size region, reach above again to press end to save.

Figured there was no need to actually hold down the selection key.  <kbd>End</kbd> also just made it easy to prevent conflicting with <kbd>Delete</kbd> when in multi region editing mode it removes the region.

**Comment by davidruhmann** [comment]
@Jaex I like your suggestion for the keys.

**Comment by Murilirum** [comment]
>Remembering alphabet keys can be more difficult for people

I chose <kbd>s</kbd> for the quick subconscious association of s = screenshot or to match the first character of the associated description.

I do however like the button functions @Jaex specified as well.

**Comment by VIIbrid** [comment]
Broken mouse user here, would it be possible to implement a setting that allows for left click to be given a start and end selection ability? My left click randomly lets go and it's much more reliable anyhow to click once to start a selection and to click a second time to end a selection. I understand this is in conflict with quick print screen, but I never use it and there's configurable keybinds for that anyhow.
